sdk/python/packages/flet-core/src/flet_core/chip.py

```diff
@@ -205,7 +205,7 @@ def before_update(self):
         self._set_attr_json("labelStyle", self.__label_style)
         self._set_attr_json("padding", self.__padding)
         self._set_attr_json("shape", self.__shape)
-        self._set_attr_json("color", self.__color)
+        self._set_attr_json("color", self.__color, wrap_attr_dict=True)
```

Comment:
I see it's been changed in multiple controls. Do you have any examples/tests to see if anything is broken by that?